### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by estebanmual**
In this milestone we: 
 - Modify the Awesome books application to have:
    - A Navigation bar.
    - Three content sections:
       - Books list.
       - Add book form.
       - Contact info.
- Make sure that when you click on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,10 +7,21 @@
...
     <main>
-      <div class="book">
+      <!-- Header Section -->
+      <header>
+        <div class="logo"><p>Awesome Books</p></div>
+        <div class="navbar">
+          <div class="links"><a href="#list" id="list-link">List</a></div>
+          <div class="links"><a href="#add" id="add-link">Add new</a></div>
+          <div class="links"><a href="#contact" id="contact-link">Contact</a></div>
+        </div>
```
Comment (rloterh) [inline index.html]: - [x] To enhance the user experience of this application kindly ensure that an **active menu link is highlighted**(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your nav menu:**
image


**The wireframe nav menu:**
image

File: index.html [hunk 2]
```diff
@@ -7,10 +7,21 @@
...
+      </header>
+      <!-- List Section -->
+      <div id="list" class="book">
         <h1>Awesome Books</h1>
         <table class="table">
```
Comment (rloterh) [inline index.html:26]: - [x] The styling of your _books list rows_ background color does not conform to the database table alternate rows color, as can be seen in the original wireframe below. Kindly ensure to improve upon this aspect.

Your design:
image

**The wireframe design:**
image

File: index.html [hunk 3]
```diff
@@ -40,6 +52,21 @@ <h2>Add new book</h2>
...
+          You can Reach out to us!
+        </p>
+        <ul>
+          <li>Our E-mail: books@awesome.com</li>
+          <li>Our Phone Number: 003311223344</li>
+          <li>Our Adress: John Doe street, 22, 883322 Vikings, Valhala</li>
+        </ul>
+      </div>
+      <!-- Footer Section -->
+      <footer><p>Copyright: All rights reserved</p></footer>
```
Comment (rloterh) [inline index.html:69]: - [x] Kindly ensure that the text `Copyright`, and is aligned to the left of the _footer_. of each page.

**Your sample page:**
image


**The wireframe pages:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Esteban & Sediq, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Site navigation is working correctly
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @Estebanmual  & @sediqullahbadakhsh,

You've done well working on the concerns from the previous review 👏🏿.
The tasks for this final milestone are now successfully implemented 💯.
All the best with your next project.

You are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**